drivers/sensor/tdk/icm42688/icm42688_common.c

```diff
@@ -255,8 +255,11 @@ int icm42688_configure(const struct device *dev, struct icm42688_cfg *cfg)
 		 * temp/accel/gyro en fields in cfg
 		 */
 		uint8_t fifo_cfg1 =
-			FIELD_PREP(BIT_FIFO_TEMP_EN, 1) | FIELD_PREP(BIT_FIFO_GYRO_EN, 1) |
-			FIELD_PREP(BIT_FIFO_ACCEL_EN, 1) | FIELD_PREP(BIT_FIFO_TMST_FSYNC_EN, 1);
+			FIELD_PREP(BIT_FIFO_TEMP_EN, 1) |
+			FIELD_PREP(BIT_FIFO_GYRO_EN, 1) |
+			FIELD_PREP(BIT_FIFO_ACCEL_EN, 1) |
+			FIELD_PREP(BIT_FIFO_TMST_FSYNC_EN, 1) |
+			FIELD_PREP(BIT_FIFO_HIRES_EN, 1);
```

Comment:
hires should probably be a config option, would be fine to set this in DT statically imho.